tests/timescaledb/README.md

```diff
@@ -0,0 +1,3 @@
+# Testing extensions
+
+* PostGIS test data from https://github.com/adityatoshniwal/postgis-sample-dataset
```

Comment:
This is incorrect, either remove the file or better add the right information.

Reply:
Sorry, I didn't look carefully. I've removed the file now.